codeserver/ubi9-python-3.11/Dockerfile.konflux.cpu

```diff
@@ -141,6 +141,10 @@ RUN echo "Installing softwares and packages" && \
     chmod -R g+w /opt/app-root/lib/python3.11/site-packages && \
     fix-permissions /opt/app-root -P
 
+RUN echo "Uninstalling gpg-pubkey RPM packages" && \
```

Comment:
@wznoinsk you're not root at this point in the dockerfile, will revisit a bit later

Reply:
modified